internal/vmcompute/vmcompute.go

```diff
@@ -544,6 +545,26 @@ func HcsGetServiceProperties(ctx gcontext.Context, propertyQuery string) (proper
 	})
 }
 
+func HcsFormatWritableLayerVhd(ctx gcontext.Context, handle uintptr) error {
+	var err error
+	ctx, span := trace.StartSpan(ctx, "HcsFormatWritableLayerVhd")
+	defer span.End()
+	defer func() {
+		if err != nil {
```

Comment:
Similar to an earlier comment, the returned error from this func needs to be named "err" in order for this defer to work.